cypress/e2e/debug_fileupload.cy.js

```diff
@@ -7,7 +7,7 @@ describe('Debug file upload', () => {
     cy.fixture('testRerun.ttr', 'binary')
       .then(Cypress.Blob.binaryStringToBlob)
       .then((fileContent) => {
-        cy.get('[data-cy-debug="upload"]').attachFile({
+        cy.get('[data-cy-debug="upload"] > input').attachFile({
```

Comment:
I recently found out that this getting a nested element is better this way, but not sure if it works with the attachFile
```suggestion
        cy.get('[data-cy-debug="upload"]').find('input').attachFile({
```

Reply:
I added the change. This would also be a good thing to add to the conventions list if we want a list of conventions for cypress.

Reply:
This is just cypress [docs](https://docs.cypress.io/api/commands/find) though